react-native/react/pinentry/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+/* @flow */
+
+import React from 'react'
+import BaseComponent from '../base-component'
+import PinentryRender from './pinentry-render'
+
+export default class Pinentry extends BaseComponent {
+  render () {
+    return <PinentryRender {...this.props} />
+  }
+
+  static parseRoute (store, currentPath, nextPath) {
+    return {
+      componentAtTop: {
+        title: 'pinentry',
+        props: {
+          onSubmit: passphrase => console.log(`Passphrase submitted: ${passphrase}`),
+          onCancel: () => console.log('Pinentry canceled'),
+          // Temp until implemented.
+          user: 'someusers'
+        }
+      }
+    }
+  }
```

Comment:
hmm I see you defined them in the render function. I guess that's okay too. Maybe you can define these props in terms of pinentry render's props?
something like:
`Pinentry.propTypes = PinentryRender.propTypes` ?